networks.json

```diff
@@ -96,7 +96,7 @@
             },
             {
               "indexed": true,
-              "name": "orderIndexes",
+              "name": "orderIds",
```

Comment:
Haven't we concluded that this would be `Index`?

Reply:
Yes, these things will be updated with the new deployment

Reply:
relevant discussion is going on here: https://github.com/gnosis/dex-contracts/pull/403

Let's make a decision and be coherent between this 2 PRs